content/work/bremtown.md

```diff
@@ -102,3 +103,100 @@ all available product options.
 {% endcall %}
 
 {% endcall %}
+
+## Brand & Logo
+
+We started with brand and logo redesign
+to establish the new look and feel
+that would reflect
+Bremtown's reimagination
+of the custom cabinetry industry.
+
+## Responsive & Accessible Website
+
+Working within the structure
+of the new brand
+as well as some established layouts and components,
+we designed a responsive website
+that meets WCAG 2.1 accessibility standards
+with CMS components
+that are user-friendly and scalable.
+
+<p style="font-size: 1.5rem;">
+Bremtown's website is responsive
+and meets WCAG accessibility standards.
+<p>
+
+We helped craft the marketing messages
+and recommended images
+to support the
+cabintry-reimagined narrative
+and to reach
+three different types
+of website visitors
+with different priorities.
+
+{% call embed.media_block(
+  media=embed.img(
+    src='work/bremtown/marketing-spec-site-responsive.png',
+    alt='Bremtown marketing site for designers to browse cabinetry products on large and small screens',
+    caption='Rebranded, accessible, and responsive marketing website with imagined marketing narrative'
+  ),
+  name='desktop-work'
+) %}
+{% endcall %}
+
+## Web Platform
+
+We created a brand new web platform
+based on Bremtown's printed spec book
+for marketing their products
+to dealers, interior designers,
+and design firms
+with the goal of establishing
+new partnerships more quickly
+and streamlining existing processes
+for current partners.
+The interactive User's Guide
+teaches new customers
+how to use the spec site
+with an intuitive, self-guided process.
+
+<p style="font-size: 1.5rem;">
+Within weeks of launch,
+Bremtown reported an increase
+in the rate of new partnerships!
+<p>
+
+Starting with wireframes,
+we iterated quickly
+based on end-user feedback.
```

Comment:
This user feedback point might be good to emphasize since they started gathering feedback early and continue to collect throughout the entire process. I am not sure how many features we built were as a result of feedback? Might be interesting to blog about one day.

Reply:
@jgerigmeyer @stacyk Yes! I would love to build the whole case study around the customer feedback and the results. I tried looking through Slack for Tim's reports about these things, but wasn't able to find anything. If Tim would be willing to give a report about how customer feedback influenced what we build and the numbers of new leads, that could be _the_ thing that helps us land similar clients.